src/stories/Seekers/CanAddSeeker/endpoint.spec.js

```diff
@@ -11,19 +11,19 @@ describe("Test API Seekers/CanAddSeeker", () => {
     };
   });
 
-  it("dummy_story_which_will_pass", async () => {
+  it("can_add_seeker_story_which_will_pass", async () => {
     let respondResult;
     try {
       const app = httpServer();
 
       const payload = {
         uuid: "e9d623ba-9507-4ae0-bf97-10b16370fec8",
         user_uuid: contextClassRef.user.user_uuid,
-        name: "Shubham Joshi",
+        name: "Amarendra Varma",
         seeker_description: {
           gender: "male",
-          age: 25,
-          experience: "4 years",
+          age: 24,
+          experience: "1/2 years",
```

Comment:
this value doesn't make sense, we won't be able to query it. we have to figure out a solution for this.